packages/terra-clinical-data-grid/src/subcomponents/Row.jsx

```diff
@@ -65,12 +67,22 @@ class Row extends React.Component {
       children,
       ...customProps
     } = this.props;
+    const theme = this.context;
+
+    const rowClass = classNames(
+      cx(
+        'row',
+        { selected: isSelected, striped: isStriped },
+        theme.className,
+        customProps.className,
```

Comment:
custom props should be defined outside of cx

Reply:
Fixed 7e45e75